acquia_lift_profiles/acquia_lift_profiles.install

```diff
@@ -180,3 +180,42 @@ function acquia_lift_profiles_update_7002(&$sandbox) {
     return t('All actions were synchronized to Lift Web');
   }
 }
+
+/**
+ * Convert existing vocabulary mappings to use taxonomy context module.
+ */
+function acquia_lift_profiles_update_7003() {
+  // 1) Convert existing mappings (if not already).
+  //
+  // Example
+  // * Before migration:
+  //   array (
+  //    'content_section' => '2',
+  //    'content_keywords' => '',
+  //    'persona' => 'taxonomy_context__country',
+  //   )
+  // * After migration:
+  //   array (
+  //    'content_section' => 'taxonomy_context__people',
+  //    'persona' => 'taxonomy_context__country',
+  //   )
+  $vocabulary_mappings = variable_get('acquia_lift_profiles_vocabulary_mappings');
+  $vocabulary_mappings_updated = array();
+  foreach ($vocabulary_mappings as $type => $vocabulary_mapping) {
+    if (empty($vocabulary_mapping)) {
+      continue;
+    }
+    $vocabulary_mappings_updated[$type] = $vocabulary_mapping;
+    if (is_numeric($vocabulary_mapping)) {
+      $vocabulary = taxonomy_vocabulary_load($vocabulary_mapping);
+      $vocabulary_mappings_updated[$type] = 'taxonomy_context' . PERSONALIZE_TARGETING_ADMIN_SEPARATOR . $vocabulary->machine_name;
+    }
+  }
+  variable_set('acquia_lift_profiles_vocabulary_mappings', $vocabulary_mappings_updated);
+
+  // 2) Enable the personalize taxonomy context module.
+  $result = module_enable(array('personalize_taxonomy_context'));
+  if (empty($result)) {
+    throw new DrupalUpdateException('Module personalize_taxonomy_context could not be enabled.');
```

Comment:
This exception will not be caught - if there is any problem enabling the module you should set an error message that the user will see after updating: 
`drupal_set_message('There was a problem enabling personalize_taxonomy module. Please ensure you have the latest Personalize module code and enable the personalize_taxonomy submodule, otherwise your vocabulary mappings will no longer work.', 'error')`